translations/translations.json

```
@@ -4860,7 +4860,13 @@
       "phoneCopied": "Telefonnummer in Zwischenablage kopiert",
       "eMailCopied": "E-Mail-Adresse in Zwischenablage kopiert",
       "websiteCopied": "Webseite in Zwischenablage kopiert",
-      "listTitle": "In deiner Nähe"
+      "listTitle": "In deiner Nähe",
+      "detailsHeader": "zurück zu Übersicht",
+      "detailsAddress": "Wegbeschreibung",
+      "detailsMapLink": "In Google Maps öffnen",
+      "detailsInformation": "Informationen",
+      "detailsNextPoi": "nächstes",
+      "detailsPreviousPoi": "vorheriges"
```

I think it should be the following as it is `Der Ort`
```suggestion
      "detailsNextPoi": "nächster",
      "detailsPreviousPoi": "vorheriger"
```